lib/components/Input/search-field/search-field.stories.tsx

```diff
@@ -0,0 +1,139 @@
+import { Meta } from "@storybook/react";
+import SearchField from ".";
+import { Status, Variants } from "../base";
+import { ComponentProps, useState } from "react";
+
+const placeholder = "Search";
+
+const variants: Record<string, Variants> = {
+    fill: "fill",
+    outline: "outline",
+};
+
+const status: Record<string, Status> = {
+    neutral: "neutral",
+    success: "success",
+    error: "error",
+};
+
+const leftStatusMessage = "Status message goes here";
+
+const meta = {
+    title: "Components/Inputs/Search Field",
+    component: SearchField,
+    parameters: {
+        layout: "centered",
+    },
+    tags: ["autodocs"],
+    args: {
+        type: "text",
+        inputSize: "md",
+        status: status.neutral,
+        disabled: false,
+        variant: variants.fill,
+        textAlignment: "left",
+        leftStatusMessage: ''
+    },
+    argTypes: {
+        type: {
+            control: {
+                type: "text",
+            },
+            options: ["text", "email", "password"],
+        },
+        inputSize: {
+            control: {
+                type: "radio",
+            },
+            options: ["sm", "md"],
+        },
+        status: {
+            control: {
+                type: "radio",
+            },
+            options: ["neutral", "success", "error"],
```

Comment:
We can also reuse status object: `Object.values(status)` or with keys

Reply:
done